modules/cnv_creating_cluster_admin_user.adoc

```diff
@@ -0,0 +1,41 @@
+[[creating-a-cluster-admin-user]]
+=== Creating a cluster admin user
+
+{ProductName} is installed across the existing
+{product-title} cluster. This requires a user with the `cluster-admin` cluster
+role. Ensure you have a user with with this cluster role, or create one
+using the default `system:admin` user on the master.
+
+.Procedure
+
+. Log in to the master as the `system:admin` user:
++
+----
+$ oc login -u system:admin
+----
+
+. Create a new user:
++
+----
+$ oc create user cnv-admin
+----
+
+. Add the `cluster-admin` cluster role to the new user:
++
+----
+$ oc adm policy add-cluster-role-to-user cluster-admin cnv-admin
+----
+
+. You can now log in as the `cnv-admin` user to install the
+{ProductName} components across the cluster:
++
+----
```

Comment:
I ran through this documentation yesterday as part of an exercise of installing the released CNV bits.  This section is confusing.  If the `cnv-admin` does not exist in the configured `identityProvider` first you cannot login because there is no configured password.  The `Additional Resources` section referencing the identity providers should be before login.

Reply:
Thanks for the comment. I've updated the content (moved existing link to a prereq and changed the step). I'd be much obliged if you could let me know if it's an improvement or not.

Reply:
@aburdenthehand so much better!